shell/app/modules/application/common/components/edit-global-variable.tsx

```diff
@@ -40,6 +40,8 @@ const convertGlobalVariableList = (globalVariable: any) => {
 };
 
 class EditGlobalVariable extends PureComponent<IEditGlobalVariableProps & FormComponentProps, any> {
+  formRef = React.createRef();
```

Comment:
I guess this ref should add generic, otherwise how to invoke `this.formRef.current.xxx()`